distribution/changelog.txt

```diff
@@ -13,7 +13,9 @@
 - Fix: [#13029] Not all Junior Roller Coaster pieces are shown when "Show all track pieces" cheat is enabled.
 - Fix: [#13044] Rides in RCT1 saves all have "0 customers per hour".
 - Fix: [#13074] Entrance and exit ghosts for mazes not being removed.
+- Fix: [#1324] Last track piece map selection still visible when placing ride entrance or exit (original bug).
```

Comment:
Could you move this line up a little, so the fixes appear in numerical order?

Reply:
oops, yep fixed now